testing/burp/01-runpath_fix.patch

```diff
@@ -0,0 +1,12 @@
+--- burp-2.1.20/Makefile.am
++++ burp-2.1.20/Makefile.am
+@@ -417,7 +417,7 @@
+ 
+ install-data-local:
+ 	$(AM_V_at)$(MKDIR_P) $(DESTDIR)$(localstatedir)/spool/$(PACKAGE_TARNAME)
+-	$(AM_V_at)$(MKDIR_P) $(DESTDIR)$(runstatedir)
++	$(AM_V_at)$(MKDIR_P) $(DESTDIR)/run
```

Comment:
This is useless, `/run` always exist on Alpine and it’s tmpfs. If program needs some directory in `/run`, it must be created in runscript, not by abuild.